### DISCUSSION TITLE
Suggestions for helper scripts (Desktop, Minecraft, Alpine)

### DISCUSSION THREAD

**Discussion by GrumpyMeow** [tteck/Proxmox#422] [Ideas]
Hi tteck,
I love the scripts you've created for Proxmox. I used some of them, but also have some made myself.
Maybe some suggestions you might want to experiment with...
* I switched over from Ubuntu/Debian templates to Alpine templates as these are way tiny. For example i based my Postfix and Caddy containers on Alpine.
* Did you know that it's possible to have a full desktop experience running in a LXC container? I think some people will greatly appreciate a user-friendly helper script.
* Some other apps for which i've created LXC containers: Spleeter, Minecraft Bedrock server

Keep up the good work!

**Comment by maddiecs84** [comment]
I've just created a [script](https://github.com/maddiecs84/Proxmox/blob/main/ct/minecraft-bedrock.sh) for installing an LXC running a Minecraft Bedrock server with auto backups (to the same disk for now).

I tried MineOS but whatever I did I could not connect to the Bedrock server. Ports were open. But nothing would connect to the actual bedrock server. At least with my script I can spin up multiple servers and not have the problem with port sharing of 19132 on the same VM.

**Reply by tteck** [reply]
That runs it as a Docker container.
